src/Commands/CommandGroup.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System.ComponentModel;
+using System.Drawing;
+using System.Security;
+using Polly;
+using Remora.Commands.Attributes;
+using Remora.Commands.Groups;
+using Remora.Discord.API.Abstractions.Objects;
+using Remora.Discord.API.Abstractions.Rest;
+using Remora.Discord.API.Objects;
+using Remora.Discord.Commands.Attributes;
+using Remora.Discord.Commands.Contexts;
+using Remora.Discord.Commands.Extensions;
+using Remora.Discord.Commands.Feedback.Services;
+using Remora.Discord.Interactivity;
+using Remora.Rest.Core;
+using Remora.Results;
+using WindowsAppCommunity.Discord.ServerCompanion.Commands.Errors;
+using NotFoundError = WindowsAppCommunity.Discord.ServerCompanion.Commands.Errors.NotFoundError;
+
+namespace WindowsAppCommunity.Discord.ServerCompanion;
+
+
+
+[Group("commands")]
+public class CommandGroup(IInteractionContext interactionContext, IFeedbackService feedbackService, IDiscordRestInteractionAPI interactionAPI, IDiscordRestChannelAPI channelApi, IDiscordRestGuildAPI guildApi, ICommandContext context) : Remora.Commands.Groups.CommandGroup
+{
+    [Command("sample-command")]
+    [SuppressInteractionResponse(true)]
+    public async Task<IResult> TestInteractivity()
```

Comment:
Let's remove this sample command back into the deleted file, so only the Portal command is here. We can rename this `CommandGroup` class to `PortalCommandGroup`.